src/tools/pub/publishing.md

```diff
@@ -236,6 +236,38 @@ follow these practices:
   Cancel the upload if any undesired files appear in that list.
 
 
+## Platform support
+
+The [pub.dev site]({{site.pub}}) detects which platforms a package supports,
+displaying these platforms on the package page.
+Users of pub.dev can filter searches by platform.
+
+Automated platform-support detection relies on the package's
+transitively imported `dart:*` libraries and
+on its Flutter plugin declarations.
```

Comment:
I find it a bit odd to mention Flutter this early, especially given we're on Dart.dev. I suggest moving that part much lower in the page.

Reply:
This is the one-sentence overview of how pub.dev detects platform support, so it seems appropriate to mention Flutter here. If we move the Flutter mention to somewhere else, then we'll need more general words here like "and other, SDK-specific fields". We could do that in the future if pub.dev adds platform-detection support for other SDKs, but it seems a shame to make it less clear for now.

Reply:
Right, but my issue is that there is a large difference between the general "guesswork" that is the library analysis, and the "declarations" that are in Flutter's platform tag. The latter is a real declaration just as the new top-level platforms tag is.

I also don't understand why we need to mention Flutter's specific platform tags here in the overview, but don't mention the new Dart tags until the fourth paragraph (just before the "Uploaders" heading.